src/CodeGen.fs

```diff
@@ -1195,12 +1231,142 @@ type GraphqlErrorResponse = {{ errors: {errorType} list }}
 type {clientName}(url: string, httpClient: HttpClient) =
     let fableJsonConverter = FableJsonConverter() :> JsonConverter
     let settings = JsonSerializerSettings(DateParseHandling=DateParseHandling.None, Converters = [| fableJsonConverter |])
+    let serializer = JsonSerializer.Create(settings)
 
     new(url: string) = {clientName}(url, new HttpClient())
+    new(httpClient: HttpClient) = {clientName}(String.Empty, httpClient)
 
     {members}"""
 
 let sampleFSharpGraphqlClient projectName clientName errorType members serializer =
     match serializer with
     | SerializerType.System -> sampleFSharpSystemGraphqlClient projectName clientName errorType members
-    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
\ No newline at end of file
+    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
+
+let sampleFableGraphqlClientFsi projectName clientName =
+        $"""namespace {projectName}
+
+open Fable.SimpleHttp
+open Fable.SimpleJson
+
+type {clientName} =
+    class
+        /// <summary>Creates {clientName} specifying list of headers</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization <b>you must</b> add
+        /// <see href="T:Fable.SimpleJson.FableJsonConverter">FableJsonConverter</see>
+        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package yourself
+        /// </remarks>
```

Comment:
These remarks are not entirely correct, the user doesn't have to add anything, it is already done within the implementation. Use this instead
```fs
/// <remarks>
/// In order to enable all F# types serialization and deserialization, this client uses Fable.SimpleJson from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a>
/// </remarks>
```